### DISCUSSION TITLE
Add missing variables from Open-Meteo

### DISCUSSION THREAD

**Discussion by EdgarOns** [breezy-weather/breezy-weather#1329] [Ideas]
### Describe your idea

Currently, different variables are offered in the application using the Open-Meteo provider. However, the Open-Meteo website shows many more meteorological variables that the API allows to add. It would be ideal to increase this data in the Breezy Weather apk and have it reflected directly on the main screen of the location for a joint and unitary view of the forecast data.

(Español)
Actualmente se ofrecen distintas variables en la aplicación usando el proveedor Open-Meteo. Sin embargo, en la web de Open-Meteo se muestran muchas más variables meteorológicas que la API permite añadir. Sería idóneo aumentar estos datos en la apk Breezy Weather y que estos se reflejasen directamente en la pantalla principal de la ubicación para una vista conjunta y unitaria de los datos del pronóstico.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Which one(s)? Currently, Open-Meteo source should support every variable Breezy Weather supports.
